backend/app/routers/routers.py

```diff
@@ -16,92 +18,99 @@
 @router.post("/login")
 async def login(
     form_data: Annotated[OAuth2PasswordRequestForm, Depends()], request: Request
-) -> Token:
+) -> AuthToken:
+    supabase_auth_service = SupabaseAuthService(settings=settings)
+
     try:
-        sign_in_result: AuthResult = auth_service.sign_in(
-            Credentials(email=form_data.username, password=form_data.password)
+        sign_in_result: AuthResponse = supabase_auth_service.sign_in(
+            Credentials(
+                email=form_data.username, password=SecretStr(form_data.password)
+            )
         )
-    except AuthInvalidCredentialsException as exception:
+    except AuthException as exception:
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
-            detail=f"Unable to login. Encountered exception with the message: {exception.message}",
+            detail=f"Unable to login. Encountered AuthException with message: {exception.message}",
             headers={"WWW-Authenticate": "Bearer"},
```

Comment:
Don't usually have this header in a response?

Reply:
I have removed `WWW-Authenticate` headers from HTTP exceptions.